lldb/tools/driver/Driver.cpp

```diff
@@ -423,6 +426,60 @@ SBError Driver::ProcessArgs(const opt::InputArgList &args, bool &exiting) {
   return error;
 }
 
+#ifdef _WIN32
+inline std::wstring GetPathToExecutableW() {
+  // Iterate until we max out the Windows max path length (256*2^7 > 32,767).
+  static const size_t maximumIterations = 7;
+  std::vector<WCHAR> buffer;
+  DWORD bufferSize = MAX_PATH;
+  for (size_t i = 0; i < maximumIterations; i++) {
```

Comment:
I wonder if this is better written as:


```suggestion
  for (size_t i = 0; buffer.size() < 32767; i++) {
```